.circleci/config.yml

```diff
@@ -54,6 +54,11 @@ jobs:
           name: run slim-lint
           command: bin/slim-lint .
 
+      # Run Sass-lint
+      - run:
+          name: run sass-lint
+          command: yarn run sass-lint -c .sass-lint.yml --no-exit -v
```

Comment:
Sass-lint default behavior is to exit on errors instead of reporting them o.O Thus requiring the `--no-exit` flag.
And without `-v` it doesn't show offenses...